### ISSUE TITLE
Dump1090 peak range capped at 269.9 nm / 500 km

### ISSUE THREAD

**Issue by Romeo-Golf** [jprochazka/adsb-receiver#298]
It appears some users are able to get valid results from in excess of 500km so wondered if it's worth either increasing the current values or providing the users with the option to increase this?

http://discussions.flightaware.com/ads-b-flight-tracking-f21/peak-range-capped-at-269-9nm-500km-t38485.html

From the looks of thinks this would be fairly trivial up allow configuration of the MAX_RANGE value in dump1090's config, any issues with increasing the dump1090_range gauge limit to say 1000km?

**Comment by Romeo-Golf** [comment]
While im not lucky enough to be seeing any packets from > 300 nm, I had a go at modifying one of my running adsb-receiver installs to confirm the required steps... and if anyone is hitting this issue would like to try the following and provide feedback that would be highly appreciated:


```
#!/bin/bash

DUMP1090_CONFIG="/etc/default/dump1090-mutability"
 grep "MAX_RANGE" ${DUMP1090_CONFIG}
 sed -e 's/^MAX_RANGE=".*"/MAX_RANGE="500"/g' -i ${DUMP1090_CONFIG}
 grep "MAX_RANGE" ${DUMP1090_CONFIG}

DUMP1090_DB="/home/pi/adsb-receiver/build/portal/graphs/dump1090.db"
 grep "dump1090_range" ${DUMP1090_DB}
 sed -e 's/value:GAUGE:0:500000/value:GAUGE:0:1000000/g' -i ${DUMP1090_DB}
 grep "dump1090_range" ${DUMP1090_DB}

DUMP1090_RRD="/var/lib/collectd/rrd/localhost/dump1090-localhost/dump1090_range-max_range.rrd"
 rrdinfo ${DUMP1090_RRD} | grep "ds\[value\].max"
 rrdtool tune ${DUMP1090_RRD} --maximum value:1000000
 rrdinfo ${DUMP1090_RRD} | grep "ds\[value\].max"
```

**Comment by Romeo-Golf** [comment]
So it appears that piaware/dump1090-fa has been configured with a value of 360nm for the last 6 months so it seems logical for the dump1090-mutablity fork to be configured with the same value:

https://github.com/flightaware/piaware-support/commit/94934c00faac4279ed2e5885fe9e919375bb5472

This looks like a good compromise between allowing users who receive signals from just over the horizon so unless there is demand for a higher default value it seems worthwhile using a consistent value across both forks :)

**Comment by jprochazka** [comment]
Added to master when code was merged from your 2.6.0 branch.
